src/@types/ApiPageHandler.ts

```diff
@@ -0,0 +1,10 @@
+import { HttpStatus } from "@fewlines/fwl-web";
+import { NextApiRequest } from "next";
+
+interface HandlerResponse {
+  status: HttpStatus;
+  data?: unknown;
+  headers?: Record<string, string>;
```

Comment:
so, either we don't put it in this PR and we'll add it when needed or we deal with it in `withAPIPageLogger` ^^

Reply:
Ohhhh damn I forgot about it..

Reply:
The only way I have to deal with it, is to pass the header to the handler, right?